### PR TITLE
Fitstools recipe

### PR DESCRIPTION
**Pull request by teake**
@conda-forge/help-c-cpp

Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/fitstools/0001_Replace_indexPos_with_indexPosInt.patch [hunk 1]
```diff
@@ -0,0 +1,69 @@
+indexPos was renamed to indexPosInt in LTL at 2010-07-29.
```
Comment (beckermr) [inline recipes/fitstools/0001_Replace_indexPos_with_indexPosInt.patch]: This patch seems like bug fix. Should it be pushed upstream?

Comment (beckermr) [reply recipes/fitstools/0001_Replace_indexPos_with_indexPosInt.patch]: I marked this one explicitly, but we should look at the others too.

**Comment by teake** [comment]
That was my plan indeed. Most of these are porting bug fixes for LTL 1.x to LTL 2.x, and I will send them to the upstream author.

**Comment by beckermr** [comment]
Great! Merging for now then. FYI (and you might already know this), but once the bug fixes are released upstream, you will have to delete the patches from the feedstock to make sure it builds. The conda-forge bots will make PRs with new versions, but you will be responsible for checking when the patches need to be deleted.

**Comment by teake** [comment]
Thanks! I will keeps tabs on this, no worries.

**Pull request merged**